packages/radio-group/theme/material/vaadin-radio-group-styles.js

```diff
@@ -25,6 +25,12 @@ const radioGroup = css`
     padding-top: 24px;
   }
 
+  /* TODO: remove when the following components are updated to use new indicator:
+  combo-box, date-picker, time-picker, date-time-picker, select. */
```

Comment:
The only remaining component is date-time-picker, please update the comment accordingly.

Reply:
Done.